packages/next/next-server/lib/utils.ts

```diff
@@ -28,6 +28,8 @@ export type AppType = NextComponentType<
   AppPropsType
 >
 
+export type AppTreeType = ComponentType<AppInitialProps>
```

Comment:
```suggestion
export type AppTreeType = ComponentType<AppInitialProps & { [name: string]: any }>
```

We should probably allow other props to be passed to `AppType`

Reply:
Done